lib/model/typing_status.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:async';
 import 'package:flutter/foundation.dart';
 
 import '../api/model/events.dart';
+import '../log.dart';
```

Comment:
nit: this happens a commit sooner than it should

(so the linter would warn at that commit)